### PR TITLE
Fixed text for email field for provider connection

### PR DESCRIPTION
**Pull request by oba2311**
Text wasn't clear - fixed it.

Only file that should be affected:

src/consts/providers/provider-credentials.ts

### CODE REVIEW

File: libs/shared/.history/src/consts/providers/provider-credentials_20220815180918.ts [hunk 1]
```diff
@@ -0,0 +1,319 @@
+import { CredentialsKeyEnum } from './provider.enum';
```
Comment (djabarovgeorge) [inline libs/shared/.history/src/consts/providers/provider-credentials_20220815180918.ts:1]: Are you aware of what are those files under .history dirictory?

File: libs/shared/src/consts/providers/provider-credentials.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ const mailConfigBase: IConfigCredentials[] = [
   {
     key: CredentialsKeyEnum.From,
     displayName: 'From email address',
-    description: 'Use the authenticated email address from the delivery provider you will send emails from.',
+    description: 'Use the same email address you used to login to your email provider',
```
Comment (djabarovgeorge) [inline libs/shared/src/consts/providers/provider-credentials.ts:8]: As I remember it the authenticated email is the one the user needs to provide there (due to the reason you can add their multiply email but the authenticated email only one).
Do you think the user will provide the relevant one if we do the change here?

Comment (scopsy) [reply libs/shared/src/consts/providers/provider-credentials.ts:8]: I agree here with @djabarovgeorge it's not the same one that is used for login. It's the authenticated email in the SMTP server of the delivery provider. They are not automatically connected. @oba2311

File: libs/shared/.history/src/consts/providers/provider-credentials_20220815181320.ts [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+];
+
+const smsConfigBase: IConfigCredentials[] = [
+  {
+    key: CredentialsKeyEnum.From,
+    displayName: 'From',
+    type: 'string',
+    required: true,
+  },
+];
```
Comment (scopsy) [inline libs/shared/.history/src/consts/providers/provider-credentials_20220815181320.ts:27]: @oba2311 Could you remove those files from the PR? Looks like they are here by mistake :)

File: libs/shared/src/consts/providers/provider-credentials.ts (hunk 2 above)
Comment (oba2311) [reply libs/shared/src/consts/providers/provider-credentials.ts:8]: @scopsy gotcha, it wasn't clear in the original sentence English a bit fuzzy - how about 
`Use the same email address you used to authenticate your email provider`

This is clear English, but I'd assume people wouldn't remember which address they used to authenticate...

File: libs/shared/.history/src/consts/providers/provider-credentials_20220815181320.ts (hunk 3 above)
Comment (oba2311) [reply libs/shared/.history/src/consts/providers/provider-credentials_20220815181320.ts:27]: yup, I probably need to exclude `.history` files... thought  those were exclude by default

File: libs/shared/src/consts/providers/provider-credentials.ts (hunk 2 above)
Comment (djabarovgeorge) [reply libs/shared/src/consts/providers/provider-credentials.ts:8]: They probebly should 
GendGrid for example

Comment (djabarovgeorge) [reply libs/shared/src/consts/providers/provider-credentials.ts:8]: They should remember it, the correct email in order to use Sendgrid for example 

image

Comment (scopsy) [inline libs/shared/src/consts/providers/provider-credentials.ts:8]:
```suggestion
    description: 'Use the same email address you used to authenticate your delivery providerr',
```
@oba2311 how about that?

**Review by scopsy — APPROVED** [review]

**Pull request merged**